ImportError: cannot import name 'AnyDevice' from 'selenium.webdriver.common.action_chains' with Selenium 4.43.0

selene 2.0.0rc9 fails to import when paired with selenium >= 4.43.0. The error occurs at startup before any test execution.

  **Error:**

  selene/core/_actions.py:30: in <module>
      from selenium.webdriver.common.action_chains import AnyDevice
  E   ImportError: cannot import name 'AnyDevice' from 'selenium.webdriver.common.action_chains'

  **Environment:**
  - Python: 3.10
  - selene: 2.0.0rc9
  - selenium: 4.43.0 (works with 4.33.0)
  - OS: macOS (Darwin 23.3.0)

  **Steps to reproduce:**

  1. Install selene==2.0.0rc9 with selenium>=4.43.0
  2. Import selene: from selene import Element
  3. ImportError is raised immediately

  **Root cause:**

  selene/core/_actions.py line 30 imports AnyDevice from selenium.webdriver.common.action_chains. This symbol was available in Selenium 4.33.0 but appears to have been removed or relocated in Selenium 4.43.0.

  Import chain:
  selene.__init__
    → selene.core._browser
      → selene.core._actions
        → selenium.webdriver.common.action_chains.AnyDevice  ← missing

  **Workaround:**

  Pin selenium to a compatible version: selenium==4.33.0

--- a/selene/core/_actions.py
+++ b/selene/core/_actions.py
@@ -27,7 +27,12 @@ from selenium.webdriver.common.actions.wheel_input import ScrollOrigin
 from selenium.webdriver.remote.webelement import WebElement
 
 from selenium.webdriver import ActionChains
-from selenium.webdriver.common.action_chains import AnyDevice
+from selenium.webdriver.common.actions.pointer_input import PointerInput
+from selenium.webdriver.common.actions.key_input import KeyInput
+from selenium.webdriver.common.actions.wheel_input import WheelInput
+
+# AnyDevice type alias for compatibility with Selenium 4.40+
+AnyDevice = Union[PointerInput, KeyInput, WheelInput]
 
 from selene.core.entity import Element
 from selene.core.configuration import Config

